src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const listOfLi = document.querySelectorAll('li');
+
+listOfLi.forEach((li) => {
+  const text = li.firstChild.data.replace(/\W/gi, '');
+
+  const span = document.createElement('span');
+
+  span.textContent = text;
+
+  li.firstChild.data = '';
+  li.insertAdjacentElement('afterbegin', span);
+});
+
+tree.addEventListener('click', (e) => {
+  const children = [...e.target.parentElement.children];
+
+  children.forEach((element) => {
+    if (
+      e.target.tagName === 'SPAN' &&
```

Comment:
Move on this condition to variable and use here
      e.target.tagName === 'SPAN' &&
      element.tagName === 'UL' &&
      element.style.display === 'none'

Reply:
Done

![image](https://github.com/mate-academy/js_sliding-tree-DOM/assets/102412814/778a3399-4b22-4c61-8aa3-7cfd8b9321c5)